repo/packages/C/confluent-kafka/10/command.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I've been trying to keep the C/confluent-kafka package aligned with the K/kafka package ... and I still saw the command.json file there.    Happy to remove it.